go/vt/mysqlctl/backup.go

```diff
@@ -352,21 +354,6 @@ func Restore(ctx context.Context, params RestoreParams) (*BackupManifest, error)
 		return nil, err
 	}
```

Comment:
looks like `ShouldStartMySQLAfterRestore` is not checked online 350-352? or it's no longer needed? the original change https://github.com/vitessio/vitess/pull/16295/files#diff-b99512e7a77c7bd6ad4dcdd037ebc57754be363e0148c67bb22886cd4fbc1d80R450

Reply:
that's a good catch, thanks Tanjin. I think this got removed because we don't exactly need it on our setup because of the hooks, but there is no reason it shouldn't be backported too. I have added it there